tests/foreman/cli/test_ansible.py

```diff
@@ -50,7 +51,8 @@ def test_positive_ansible_e2e(target_sat, module_org, rhel_contenthost):
     :CaseImportance: Critical
     """
     SELECTED_ROLE = 'RedHatInsights.insights-client'
-    SELECTED_VAR = 'insights_variable'
+    SELECTED_ROLE_1 = 'theforeman.foreman_scap_client'
```

Comment:
I think it would be nice if we use the ROLES list here instead of two such vars similar to the host group test and follow the same scenario of assign/add/remove?

Reply:
Assign/add/remove all three functionalities are implemented, I did not create the list here because I just wanted one extra role to play around these functionalities and the one for running the ansible job.

Reply:
A non blocking comment !

Reply:
I think its fine without creating the list of roles :)